test/RawDataset.js

```diff
@@ -280,6 +283,26 @@ describe("1900: RawDataset: Raw Datasets", () => {
     );
   });
 
+  it("01250: adds a new proposal for pattching in the existing dataset", async () => {
```

Comment:
fix test string to "0124: adds a new proposal for patching in the existing dataset"